src/cpdb/cpdb.py

```diff
@@ -1,18 +1,23 @@
 import streamlit as st  # type: ignore
 import pandas as pd
 from src.cpdb.helpers import (
+    REPO_NAME,
+    BUCKET_NAME,
     get_data,
     get_geometries,
     get_commit_cols,
     get_diff_dataframe,
     get_map_percent_diff,
     sort_base_on_option,
     VIZKEY,
-    cpdb_published_versions,
+    #cpdb_published_versions,
```

Comment:
Can probably delete this, but just left for now. Hard-coded previously which versions you could compare to